cluster-autoscaler/utils/drain/drain_test.go

```diff
@@ -461,6 +496,24 @@ func TestDrain(t *testing.T) {
 			expectPods:          []*apiv1.Pod{failedPod},
 			expectDaemonSetPods: []*apiv1.Pod{},
 		},
+		{
+			description: "pod in unknown state - skipTerminatingPods",
+			pods:        []*apiv1.Pod{unknownStatePodWithDeletionTimestamp},
+			pdbs:        []*policyv1.PodDisruptionBudget{},
+			rcs:         []*apiv1.ReplicationController{&rc},
+			expectFatal: false,
+			expectPods:  []*apiv1.Pod{},
+			expectDaemonSetPods: []*apiv1.Pod{},
+		},
+		{
+			description: "pod in unknown state with extended gracePeriod - skipTerminatingPods",
```

Comment:
Could you make the description more accurate? E.g. something like "long terminating pod with extended grace period".

Reply:
done